### DISCUSSION TITLE
Could it be possible for players to copy the entire Customization Layers Breakdown and paste it to another copy of the same weapon?

### DISCUSSION THREAD

**Discussion by MrPigZ** [HelixNebulaStudio/RiseOfTheDead#624] [Ideas]
For example let's say that I have awesome looking desolator and someone else also wants same looking desolator. I could either show them the breakdown or take a picture of every part. Both of these methdots have major downsides.

Could it be possible to copy all of the customization settings and paste them to a weapon?

This would also add a feature which people have wanted quite some time now. A way to have multiple customization for 1 weapon. If this feature was added they could copy the settings save it on their pc and then make a new skin and save that too.

**Comment by MXKhronos** [comment]
This is planned for the future.
